modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -39,16 +39,30 @@ export default class NestedTabs extends Base {
 		return tabTitleElement.getAttribute( 'data-tab-index' );
 	}
 
+	getActiveTabIndex() {
+		const settings = this.getSettings(),
+			activeTitleFilter = settings.ariaAttributes.activeTitleSelector,
+			$activeTitle = this.elements.$tabTitles.filter( activeTitleFilter );
+
+		return $activeTitle.attr( 'data-tab-index' ) || null;
```

Comment:
`data-tab-index` can be called from `getDefaultSettings()`

Reply:
Updated.